pkg/work/hub/controllers/manifestworkreplicasetcontroller/manifestworkreplicaset_deploy_reconcile.go

```diff
@@ -278,8 +278,8 @@ func CreateManifestWork(mwrSet *workapiv1alpha1.ManifestWorkReplicaSet, clusterN
 				ManifestWorkReplicaSetPlacementNameLabelKey:  placementRefName,
 			},
 			Annotations: map[string]string{
-				common.CloudEventsDataTypeAnnotationKey:   payload.ManifestBundleEventDataType.String(),
-				common.CloudEventsGenerationAnnotationKey: fmt.Sprintf("%d", mwrSet.Generation),
+				common.CloudEventsDataTypeAnnotationKey:        payload.ManifestBundleEventDataType.String(),
```

Comment:
could you explain why these annotation is needed? should it be optionally set?

Reply:
add the comments, these are only needed by cloudevents client

Reply:
only add them when cloudevents client is used

Reply:
removed